lua/entities/gmod_wire_light.lua

```diff
@@ -244,20 +250,28 @@ function ENT:Setup(directional, radiant, glow, brightness, size, r, g, b, sprite
 		self.spritesize = math.Clamp( self.spritesize, 0, 256 )
 	end
 
+	if not self.on then self:SetOn(true) end
+	
 	self:Directional( self.directional )
 	self:Radiant( self.radiant )
 	self:SetGlow( self.glow )
 	self:SetBrightness( self.brightness )
 	self:SetSize( self.size )
 	self:SetSpriteSize( self.spritesize )
 
+	local inputs = {"Red", "Green", "Blue", "RGB [VECTOR]", "GlowBrightness", "GlowSize", "SpriteSize"}
 	if self.glow then
-		WireLib.AdjustInputs(self, {"Red", "Green", "Blue", "RGB [VECTOR]", "GlowBrightness", "GlowSize", "SpriteSize"})
-	else
-		WireLib.AdjustInputs(self, {"Red", "Green", "Blue", "RGB [VECTOR]", "SpriteSize"})
+		table.insert(inputs, 5, "GlowBrightness")
+		table.insert(inputs, 6, "GlowSize")
 	end
+	
+	if self.on then
+		table.insert(inputs, 1, "On")
+	end
+	
+	WireLib.AdjustInputs(self, inputs)
 
 	self:UpdateLight()
 end
 
-duplicator.RegisterEntityClass("gmod_wire_light", WireLib.MakeWireEnt, "Data", "directional", "radiant", "glow", "brightness", "size", "R", "G", "B", "spritesize")
+duplicator.RegisterEntityClass("gmod_wire_light", WireLib.MakeWireEnt, "Data", "on", "directional", "radiant", "glow", "brightness", "size", "R", "G", "B", "spritesize")
```

Comment:
New stuff needs to be put at the end of the list or backwards compatibility will be broken